### PR TITLE
add param value for 'tidb_replica_read'

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2977
- Other reference link(s):

### CODE REVIEW

File: follower-read.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ To enable TiDB's Follower Read feature, set the value of the `tidb_replica_read`
 {{< copyable "sql" >}}
 
 ```sql
-set @@tidb_replica_read = 'follower';
+set @@tidb_replica_read = 'target value';
```
Comment (yikeke) [inline follower-read.md:27]:
```suggestion
set @@tidb_replica_read = '<target value>';
```

File: follower-read.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ This variable is used to set the data read mode expected by the current session.
 - When the value of `tidb_replica_read` is set to `leader` or an empty string, TiDB maintains its original behavior and sends all read operations to the leader replica to perform.
 - When the value of `tidb_replica_read` is set to `follower`, TiDB selects a follower replica of the Region to perform all read operations.
 - When the value of `tidb_replica_read` is set to `leader-and-follower`, TiDB can select any replicas to perform read operations.
+- When the value of `tidb_replica_read` is set to `leader-and-follower`, read requests are load-balanced between the leader and followers.
```
Comment (yikeke) [inline follower-read.md:39]:
```suggestion
```
PLEASE pay attention to the context. @TomShawn line 38 already covers your new change.

File: follower-read.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ To enable TiDB's Follower Read feature, set the value of the `tidb_replica_read`
 {{< copyable "sql" >}}
```
Comment (yikeke) [inline follower-read.md:24]: Please modify [line 22](https://github.com/pingcap/docs/pull/3122/files#diff-a2c19cd7e1403c5a9e29745e800b4b59R22): 

To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` session variable:

Comment (TomShawn) [reply follower-read.md:24]: Done. PTAL.

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -372,10 +372,11 @@ The Plan Cache configuration of the `PREPARE` statement.
 
 - The number of cached statements.
 - Default value: `100`
+- The type is `uint`. Values less than 0 are converted to large integers.
```
Comment (yikeke) [inline tidb-configuration-file.md:375]: This is aligned in https://github.com/pingcap/docs/pull/3123/files#diff-c8655fccf1cdae606dc717101a6cf6b3R375

**Pull request merged**